### PR TITLE
Update fritzing

### PR DESCRIPTION
**Pull request by i0ntempest**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/fritzing.rb [hunk 1]
```diff
@@ -1,28 +1,17 @@
 cask 'fritzing' do
-  version '0.9.3b'
-  sha256 'a057ed849b842540b98a68ab2cb996e22b482278706dd2f8da21d1bccf70513f'
+  version '0.9.4,a1ffcea08814801903b1a9515b18cf97067968ae:498'
+
+  if MacOS.version >= '10.14'
```
Comment (vitorgalvao) [inline Casks/fritzing.rb]: In conditionals, always use release names (i.e. `:mojave`) and not numbers, and always do `<=`, with the last branch being an `else`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Thank you for the fast turnaround.

**Comment by core-code** [comment]
ping @vitorgalvao 

the line is faulty:
`  if MacOS.version <= ':high_sierra'`

:high_sierra isn't supposed to be in quotes, and isn't quoted in any other cask. i'll send a PR to correct this shortly

i think travis should catch that case if would be easy to do so @reitermarkus

**Comment by core-code** [comment]
https://github.com/Homebrew/homebrew-cask/pull/73552

**Pull request merged**